packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/__tests__/quanticRefineModalContent.test.js

```diff
@@ -286,46 +387,341 @@ describe('c-quantic-refine-modal-content', () => {
+          selectors.quanticSortOption
+        );
+
+        expect(quanticSortOption).not.toBeNull();
+        Object.entries(exampleSortOptionAttributes).forEach(([key, value]) => {
+          expect(quanticSortOption[key]).toEqual(value);
+        });
+      });
+    });
+
+    describe('the clear all filters button', () => {
+      describe('when there are active filters', () => {
+        beforeEach(() => {
+          setupMockFacetsDataInQuanticStore({
+            filetype: {
+              label: 'File Type',
+              facetId: 'filetype',
+              element: {
+                localName: 'c-quantic-facet',
+                ...exampleFacetAttributes,
+              },
+              metadata: {
+                customCaptions: [],
+              },
+            },
+          });
+          breadcrumbManagerState = {
+            ...breadcrumbManagerState,
+            hasBreadcrumbs: true,
+          };
+        });
+
+        it('should display the clear all filters button', async () => {
+          const element = createTestComponent();
+          await flushPromises();
+
+          const clearActiveFiltersButton = element.shadowRoot.querySelector(
+            selectors.clearActiveFiltersButton
+          );
+
+          expect(clearActiveFiltersButton).not.toBeNull();
+        });
+      });
+
+      describe('when there are no active filters', () => {
+        beforeEach(() => {
+          setupMockFacetsDataInQuanticStore({
+            filetype: {
+              label: 'File Type',
+              facetId: 'filetype',
+              element: {
+                localName: 'c-quantic-facet',
+                facetId: 'filetype',
+              },
+              metadata: {
+                customCaptions: [],
+              },
+            },
+          });
+          breadcrumbManagerState = {
+            ...breadcrumbManagerState,
+            hasBreadcrumbs: false,
+          };
+        });
+
+        it('should not display the clear all filters button', async () => {
+          const element = createTestComponent();
+          await flushPromises();
+
+          const clearActiveFiltersButton = element.shadowRoot.querySelector(
+            selectors.clearActiveFiltersButton
+          );
+
+          expect(clearActiveFiltersButton).toBeNull();
+        });
+      });
+    });
+
+    describe('the filters title', () => {
+      beforeEach(() => {
+        setupMockFacetsDataInQuanticStore({
+          filetype: {
+            label: 'File Type',
+            facetId: 'filetype',
+            element: {
+              localName: 'c-quantic-facet',
+              ...exampleFacetAttributes,
+            },
+            metadata: {
+              customCaptions: [],
+            },
+          },
+        });
+        breadcrumbManagerState = {
+          ...breadcrumbManagerState,
+          hasBreadcrumbs: true,
+        };
+      });
+
+      describe('when hideSort is set to false and a facet is rendered', () => {
```

Comment:
```suggestion
      describe('when #hideSort property is set to false and a facet is rendered', () => {
```

similar to my other comment